internal/store/store.go

```diff
@@ -302,8 +313,9 @@ func PrivateAuthenticatedPreset(c Config) Option {
 }
 
 // PrivateUnauthenticatedPreset is the default Option when connecting to the private API without authentication.
-func PrivateUnauthenticatedPreset(c Config) Option {
-	options := []Option{Service(c.Service())}
+func PrivateUnauthenticatedPreset(c BasicConfig) Option {
+	// Default to cloud
+	options := []Option{Service(c.Service()), WithBaseURL(atlas.CloudURL)}
```

Comment:
[q] Was the missing default URL  the reason why we didn’t get the public IP address of Melissa?

Reply:
correct, it was using the full path `http://cloud.mongodb.com/public/api/v1` and for these endpoints we can't